src/styles/getModalStyles.js

```diff
@@ -154,7 +154,7 @@ export default (type, windowDimensions, popoverAnchorPosition = {}, containerSty
                 borderColor: themeColors.border,
                 justifyContent: 'center',
                 overflow: 'hidden',
-                boxShadow: '0px 0px 10px 0px rgba(0, 0, 0, 0.025)',
+                boxShadow: '0px 4px 12px 0px rgba(0, 0, 0, 0.06)',
```

Comment:
I wonder if we should store this as some kind of global style/variable in case we want to use it elsewhere.

Reply:
Sounds good.